coverage/.last_run.json

```diff
@@ -1,5 +1,5 @@
 {
```

Comment:
Not really sure why coverage is still pushing up to github. Tried to implement .gitignore within this branch to hopefullt resolve this issue!